### DISCUSSION TITLE
jekyll-theme-satellite/visitor%20counter

### DISCUSSION THREAD

**Discussion by giscus** [byanko55/jekyll-theme-satellite#47] [Q&A]
# jekyll-theme-satellite/visitor%20counter

Satellite - jekyll blog theme

https://byanko55.github.io/jekyll-theme-satellite/visitor%20counter.html

**Comment by joshbushdash** [comment]
Hi, I've tested that the counter is only activated when people click in the blogs.  {% if page_type == "post" %}
Is it more useful that the home page is also recorded?
